uint8 RAMP=1
uint8 SINOID=2
uint8 profile_type

float64 vel
float64 accl
